pyproject.toml

```diff
@@ -114,6 +114,7 @@ cohere = { version = "^4.56", optional = true }
 pytest = { version = "^7", optional = true }
 pytest-asyncio = { version = "^0.23.0", optional = true }
 mock = { version = "^5", optional = true }
+slack-bolt = "^1.20.1"
```

Comment:
please set this dependency as optional, you can refer to how other tools are added